### DISCUSSION TITLE
Docker images in docker-compose deprecated or private. Is there a public docker repo?

### DISCUSSION THREAD

**Discussion by bitodt** [crypto-org-chain/chain-main#434] [Q&A] [answered]
**Describe the bug**
docker-compose.yaml uses the `cryptocom/chainmaindnode` image, which gives the following error when pulled:

> Error response from daemon: pull access denied for cryptocom/chainmaindnode, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

**To Reproduce**
```
git clone https://github.com/crypto-org-chain/chain-main.git
cd chain-main
docker-compose up
```

**Expected behavior**
A docker setup is spin-up

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - OS: Debian Buster

**Additional context**
Is there a public dockerhub image to use? If no, docker-compose should build from context.

**Comment by tomtau** [comment]
@yihuang @lezzokafka @allthatjazzleo AFAIK no public docker image, but one can easily build one given the binaries are statically linked?

**Reply by akamsmrhill** [reply]
I wish I knew more/ I've tried a couple places to get local help. Any suggestions?

**Comment by tomtau** [comment, marked as answer]
this is tracked under this issue: https://github.com/crypto-org-chain/chain-docs/issues/128
